airgo/refresh_data.go

```diff
@@ -0,0 +1,98 @@
+package airgo
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	airtable "github.com/mottaquikarim/go-airtable"
+	log "github.com/sirupsen/logrus"
+)
+
+type RefreshOpts struct {
```

Comment:
Lines 13-17 we create structs in order to define the variables all under one block of memory